torchvision/models/detection/transform.py

```diff
@@ -208,7 +209,7 @@ def resize_keypoints(keypoints, original_size, new_size):
 
 def resize_boxes(boxes, original_size, new_size):
     # type: (Tensor, List[int], List[int])
-    ratios = [float(s) / float(s_orig) for s, s_orig in zip(new_size, original_size)]
+    ratios = [torch.scalar_tensor(s) / torch.scalar_tensor(s_orig) for s, s_orig in zip(new_size, original_size)]
```

Comment:
you probably need to specify the dtype as `float32` otherwise we might have an issue with the integer division